### ISSUE TITLE
select current line keyboard shortcut gone

### ISSUE THREAD

**Issue by zarkle** [Microsoft/vscode#70114]
- VSCode Version: 1.32.0 (1.32.0)
- OS Version: 10.13.6 (High Sierra)

I constantly use cmd-I to select the current line and it suddenly stopped working today after the latest release. Please put it back!  I don't see an option to add it in the Keyboard Shortcuts either.  It's even on the Keyboard shortcuts PDF (https://code.visualstudio.com/shortcuts/keyboard-shortcuts-macos.pdf).

**Comment by ewolfe** [comment]
It's `Cmd+L` now. I was pretty upset about this as well. A notice would've been nice.

https://code.visualstudio.com/updates/v1_32#_default-keybinding-change-for-expandlineselection

**Comment by doggy8088** [comment]
@ewolfe I'm using Windows.  The `Ctrl+L` can't trigger `expandLineSelection`.  If I change keybinding to `Ctrl+I` then it works.  Do you know why?

**Comment by VictorBaudot** [comment]
Neither Cmd+I or Cmd+L works for me

**Comment by ewolfe** [comment]
@doggy8088 I don't know. I haven't used Windows in quite some time. You might have something else already mapped to `Ctrl L` - either at a higher system level or within VSCode itself. I would recommend searching for `Ctrl L` in your keyboard shortcut preferences (in VS Code).

**Comment by doggy8088** [comment]
I restarted my laptop and VSCode. It works now. Don't know why. 😝

**Comment by alexandrudima** [comment]
@doggy8088 Can you please check what you have bound to Ctrl+L?

![keyboard-conflicts](https://cloud.githubusercontent.com/assets/5047891/25484174/38f5b004-2b59-11e7-994b-dfa96ef043ce.gif)

**Comment by doggy8088** [comment]
@alexandrudima I do have a existing `Ctrl+L` bound to a command, but the `when` are limited to `markdown` language only.  So this could possibly a bug in VSCode somewhere.  No matter how, my VSCode is ok now. I don't know why.

**Comment by benjaminmillhouse** [comment]
Same issue.  Please put it back.  You shouldn't just replace/remove a widely used keybinding...

**Comment by McDaddy** [comment]
I don't think cmd + L can perform the same action as cmd + I

**Comment by HughGrovesArup** [comment]
When I use "ctrl + l" I get this message:

![image](https://user-images.githubusercontent.com/40033911/54277106-d5142900-4571-11e9-9543-6f267d1211dc.png)

However in my key bindings the only shortcut for "ctrl + l" is expand line selection

![image](https://user-images.githubusercontent.com/40033911/54277135-eb21e980-4571-11e9-8f76-cfd5328639a0.png)

Quite an annoying bug!

**Comment by doggy8088** [comment]
@HughGrovesArup YES, this message is what I was facing few days ago. `(Ctrl+L) was pressed. Waiting for second key of chord...`.

**Comment by trmatt** [comment]
I also had the same issue as @HughGrovesArup and @doggy8088 with the Default keybinding.

When I set the keybinding to CTRL+I, it works (as a User binding).

Then, setting it back to CTRL+L, it continues to work (as a User binding).

**Comment by alexandrudima** [comment]
It is not really helpful to keep this issue open, as I plan no further changes in this area.

**TL;DR** Starting with 1.32, `expandLineSelection` is now bound to <kbd>Ctrl</kbd>+<kbd>L</kbd> and no longer to <kbd>Ctrl</kbd>+<kbd>I</kbd>. Out of the box, there is no conflict and it works under <kbd>Ctrl</kbd>+<kbd>L</kbd>.

There is an extension out there (perhaps a keymap extension, I don't know exactly which one) that binds show output to a chord which starts with <kbd>Ctrl</kbd>+<kbd>L</kbd>. If you have that extension, please let me know which one it is, so I can reach out to the extension author.

To unblock yourself, you can edit the keybindings and search for `expandLineSelection` and bind it back to <kbd>Ctrl</kbd>+<kbd>I</kbd>.
